bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_utils.py

```diff
@@ -11,21 +11,30 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.                    #
 ################################################################################
 
-import cppyy
-
 
 def _getter(k, v):
     # helper function to get CmdArg attribute from `RooFit`
     # Parameters:
     # k: key of the kwarg
     # v: value of the kwarg
+
+    # We have to use ROOT here and not cppy.gbl, because the RooFit namespace is pythonized itself.
+    import ROOT
+
+    func = getattr(ROOT.RooFit, k)
+
+    if func.__class__.__name__ == "CPPOverload":
+        if "()" in func.func_doc:
```

Comment:
Why this?

Reply:
@etejedor Sir, that is to check for functions that don't pass any arguments

Reply:
@etejedor Sir, that is in order to check for functions that don't pass any RooCmdArgs like ShiftToZero and MoveToBack

Reply:
I think from this answer it's still not very clear for someone who is not that familiar with RooFit. I think it's better to even add some comments to the code explaining what this is about. Explaining with an example is always good too, I would use this example:
```Python
pdf.plotOn(frame, ROOT.RooFit.MoveToBack())
```
Then you explain why this needs special consideration in the keyword argument pythonization and what you did in this PR to enable this syntax:
```Python
pdf.plotOn(frame, MoveToBack=True)
```

Reply:
@guitargeek Sir, Should we add docstring instead of comment?

Reply:
This is more a place for comments, because it's to explain a very local piece of code to developers that might read it.